docs/actions.md

```diff
@@ -0,0 +1,10 @@
+# Actions that can be redefined by the user
+
+|        | name                  | working directory | when run                                                                          | description                               |
+| ------ | --------------------- | ----------------- | --------------------------------------------------------------------------------  | ----------------------------------------- |
+| [hook] | `init`                | upstream git repo | after the cloning of the upstream-repo (master) and before other operation        |                                           |
+| [hook] | `pre-sync`            | upstream git repo | after the cloning of the upstream-repo and checkout to the right (release) branch |                                           |
```

Comment:
I'd remove the `the` where used with a verb, i.e. `after cloning` instead of `after the cloning`.